docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -2158,6 +2158,12 @@ Available in Grafana v9.5.0 or later, and [OpenTelemetry must be configured as w
 
 If `true`, propagate the tracing context to the plugin backend and enable tracing (if the backend supports it).
 
+## as_external
+
+Load an external versions of a core plugin. The external version needs to be explicitly installed.
```

Comment:
```suggestion
Install an external version of a core plugin explicitly.
```

Reply:
This would be changing the meaning. Just setting this flag won't install the plugin, it will just load the external version if it has been installed already.

Reply:
Thanks for the feedback! Looks good to me.